### PR TITLE
Update Portuguese translation for 1.22.1

### PR DESCRIPTION
**Pull request by liddack**

### CODE REVIEW

File: resources/pt/cemu.po [hunk 1]
```diff
@@ -2230,6 +2245,10 @@ msgstr "Coreinit &Memory API"
 msgid "&Coreinit MP API"
 msgstr "Coreinit M&P API"
 
+#: MainWindow.cpp
+msgid "&Coreinit Thread API"
+msgstr "Coreinit M&P API"
```
Comment (Fs00) [inline resources/pt/cemu.po:2250]: A small copy-paste issue :)

**Review by Fs00 — CHANGES_REQUESTED** [review]
There's just a small issue to fix before merging.

**Comment by liddack** [comment]
Oops... Now it's fixed. Thanks!

**Review by Fs00 — APPROVED** [review]

**Comment by Fs00** [comment]
Thanks!

**Pull request merged**